python-package/src/atls/httpa_connection.py

```diff
@@ -1,8 +1,9 @@
-import socket
-from http.client import HTTPS_PORT, HTTPConnection
-from typing import Optional, Tuple
+from typing import ClassVar, Optional, Tuple
 
 from atls import ATLSContext
+from urllib3.connection import HTTPConnection, port_by_scheme
+from urllib3.util.connection import _TYPE_SOCKET_OPTIONS
+from urllib3.util.timeout import _DEFAULT_TIMEOUT, _TYPE_TIMEOUT
```

Comment:
Yeah, I don't like it either. The alternative though is duplicating them, which felt nastier when I was considering what to do.

Thinking about it more now, maybe I should duplicate them since these end up being part of the aTLS package's public API.